index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Diego Vidal</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet"> -->
+
+</head>
+
+<body>
+    <div class="body-wrap">
+
+        <div class="nav-flex-container">
+            <div class="nav-logo">My Logo</div>
+            <button class="nav-menu-button"><img class=" nav-logo" src="./icons/Icon - Menunav-button.png"
+                    alt="navegation button"></button>
+            <p>Portfolio</p>
+            <p>About</p>
+            <p>Contact</p>
+        </div>
+
+        <header class="header-container">
+
+            <section class="header-text">
+                <h1>
+                    I’m Alexandra. <br>
+                    Glad to see you!
+                </h1>
+                <p>
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and
+                    experience! If you like what you see and have a project
+                    you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="lets-conect">
+                    LET'S CONNECT
+                </p>
+                <div class="social-icon-container">
+                    <button class="twitter-button"><img src="./icons/tw-icon.png" alt="twitter button"></button>
+                    <button class="linkedin-button"><img src="./icons/Linkedin iconin-icon.png"
+                            alt="linked in button"></button>
+                    <button class="m-button"><img src="./icons/m-icon.png" alt="m button"></button>
+                    <button class="github-button"><img src="./icons/github-icon.png" alt="github button"></button>
+                    <button class="hand-button"><img src="./icons/hand-icon.png" alt="a hand button"></button>
+                </div>
```

Comment:
Since these will serve as a way for potential clients to contact you, I encourage you to use an `a` tag to wrap each icons and also, you will agree with me that they are a list of icons, and i think it is nice to put them inside of a `ul` tag.

Like this

```
<ul>
  <li><a>icon</a></li>
</ul>
```